<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AERPEditConnectOptionsDlg</class>
 <widget class="QWidget" name="AERPEditConnectOptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlephERP - Configuración de conexión</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/aplicacion/images/alephLogoIcon.ico</normaloff>:/aplicacion/images/alephLogoIcon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gbGeneral">
     <property name="title">
      <string>Opciones de conexión</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Nombre de la conexión</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leName"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Driver de conexión</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbType">
        <item>
         <property name="text">
          <string>Nativo (PostgreSQL)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ODBC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SQLite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cloud (Nube)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Firebird / Interbase</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Dirección del servidor</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leServer"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Puerto</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lePort"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Base de datos</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leDatabase"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Usuario de base de datos</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="leUser"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lePassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Opciones de conexión</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="leConnectionOptions"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Protocolo</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="leCloudProtocol"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>DSN</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leODBC"/>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>165</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>155</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="gbOther">
     <property name="title">
      <string>Otros parámetros de conexión</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Esquema de base de datos</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leSchema"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Prefijo de tablas de sistema</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leSystemTables"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pbSave">
     <property name="text">
      <string>&amp;Save</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/saveAndClose.png</normaloff>:/generales/images/saveAndClose.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="gbCloud">
     <property name="title">
      <string>Datos conexión al sistema remoto</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Nº Licencia</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Usuario de acceso</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leCloudUser"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leCloudPassword"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leLicenseKey"/>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tras su registro en AlephERP Web, se le han proporcionado tres datos:&lt;/p&gt;&lt;p&gt;· Un usuario de acceso a la base de datos en la nube&lt;/p&gt;&lt;p&gt;· Una contraseña para ese usuario&lt;/p&gt;&lt;p&gt;· Un código de licencia única para usted.&lt;/p&gt;&lt;p&gt;Debe introducirlos a continuación.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>cbType</tabstop>
  <tabstop>leServer</tabstop>
  <tabstop>lePort</tabstop>
  <tabstop>leDatabase</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leConnectionOptions</tabstop>
  <tabstop>leCloudProtocol</tabstop>
  <tabstop>leCloudUser</tabstop>
  <tabstop>leCloudPassword</tabstop>
  <tabstop>leLicenseKey</tabstop>
  <tabstop>leSchema</tabstop>
  <tabstop>leSystemTables</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>leODBC</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
